### PR TITLE
docs: exclude deprecated files from broken url check DOC-1503

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR skips all files in the deprecated folder from broken URL checks, as we no longer wish to maintain them. 

## Changed Pages

No user facing changes.

## Jira Tickets

🎫 [DOC-1503](https://spectrocloud.atlassian.net/browse/DOC-1503)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._

[DOC-1503]: https://spectrocloud.atlassian.net/browse/DOC-1503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,12 @@ ALOGLIA_CONFIG=$(shell cat docsearch.dev.config.json | jq -r tostring)
 # Find all *.md files in docs, cut the prefix ./ 
 # Remove all security-bulletins and cve-reports.md because they are rate limited by nvd.nist.gov
 # Remove oss-licenses.md because they are rate limited by npmjs.com
-VERIFY_URL_PATHS=$(shell find ./docs -name "*.md" | cut -c 3- | sed '/security-bulletins/d' | sed '/cve-reports/d' | sed '/oss-licenses/d')
+# Remove all /deprecated paths because we don't want to maintain their links
+VERIFY_URL_PATHS=$(shell find ./docs -name "*.md" | cut -c 3- | \
+	sed '/security-bulletins/d' | \
+	sed '/cve-reports/d' | \
```
Comment (karl-cardenas-coding) [inline Makefile:21]: The security bulletins have this folder structure `/security-bulletins/reports/pc-cve-2023-24536`.  I don't think we  we need `/cve-reports` anymore.

**Review by benradstone — APPROVED** [review]
Looks good to me, thanks for explaining this.

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good just one minor question

File: Makefile (hunk 1 above)
Comment (addetz) [reply Makefile]: Sorted now! Thank you.

**Pull request merged**